secretsmanager/src/aes.h

```diff
@@ -22,21 +22,33 @@
 #include <cryptopp/modes.h>
 #include <cryptopp/aes.h>
 
+#include <openssl/ec.h>
+#include <openssl/evp.h>
+#include <openssl/pem.h>
+#include <openssl/aes.h>
+
 #include "key_params.h"
 
 namespace concord::secretsmanager {
 
+using std::string;
+using std::vector;
+using std::unique_ptr;
+
 class AES_CBC {
   CryptoPP::AES::Encryption aesEncryption;
   CryptoPP::AES::Decryption aesDecryption;
   CryptoPP::CBC_Mode_ExternalCipher::Encryption enc;
   CryptoPP::CBC_Mode_ExternalCipher::Decryption dec;
 
+  vector<uint8_t> key;
```

Comment:
we either need CryptoPP::* objects or key and iv (right)?
I do not see any compilation flag here.

Reply:
Flags added.